src/Umbraco.Tests/Services/ContentTypeServiceTests.cs

```diff
@@ -1078,12 +984,13 @@ public void Can_Copy_ContentType_By_Performing_Clone()
             var metaContentType = MockedContentTypes.CreateMetaContentType();
             service.Save(metaContentType);
 
-            var simpleContentType = MockedContentTypes.CreateSimpleContentType("category", "Category", metaContentType);
+            var simpleContentType = MockedContentTypes.CreateSimpleContentType("category", "Category", metaContentType) as IContentType;
```

Comment:
Shouldn't use `as ` casting unless you are expecting it not to cast and return null. Better to explicitly cast since you know what you are expecting otherwise you'll end up with NullRef exceptions